### PR TITLE
Day/00/phase01

### PR DESCRIPTION
**Pull request by fxbabin**
First pull request.

### CODE REVIEW

File: day00/day00.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Bootcamp Data Engineering
+
+# Day00 - SQL with PostgreSQL
+
+Today, you will learn how to use a SQL database: PostgreSQL.
+
+## Notions of the day
+
+The purpose of the day is first to create, administrate and normalize a PostgreSQL Database. Then, we are going to analyse the data and visualize the content of the database. Finally, we will see advanced notions like caching, replication and backups.
```
Comment (madvid) [inline day00/day00.md:9]: The purpose of the day is ** at ** first to create, (il me semble)

File: day00/ex03/ex03.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+![tables](../imgs/tables.png){width=450px}
+
+nb : Foreign keys are a reference to an existing column in another table.
+
+2) You will have to create the 3 populate functions
+
+* **populate_appstore_games**
+* **populate_appstore_games_genres**
+* **populate_appstore_games_languages**
```
Comment (madvid) [inline day00/ex03/ex03.md:37]: It is not important but the functions are in bold here but not before on lines 23 to 25, the format you do here is then slightly different

File: day00/ex03/psycopg2_basics.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+Tables can be created with the cursor.
+```python
+curr.execute("""CREATE TABLE members (
+                    id serial PRIMARY KEY,
+                    firstname varchar(32),
+                    lastname varchar(32),
+                    birthdate date
+                )
+            """)
```
Comment (madvid) [inline day00/ex03/psycopg2_basics.md:40]: I don't know if the fact it is on several lines is done on purpose

Comment (madvid) [reply day00/ex03/psycopg2_basics.md:40]: I just see the following, I understand it is done on purpose then

File: day00/ex07/ex07.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex07              |
+|   Files to turn in :    |  seniors.py        |
+|   Forbidden functions : |  None              |
+|   Remarks :             |  n/a               |
+
+## Objective
+
+You must show the Name of developpers involved in games released before 01/08/2008 included and update after 01/01/2018 included.
```
Comment (madvid) [inline day00/ex07/ex07.md:12]: developers* (only one 'p' in english)

File: day00/ex07/ex07.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Instructions
+
+You must create a program using a function `get_seniors` that shows the Name of developpers involved in games released before 01/08/2008 included and update after 01/01/2018 included.
```
Comment (madvid) [inline day00/ex07/ex07.md:16]: developers* (only one 'p' in english)

File: day00/ex12/ex12.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+|   Forbidden function :  |  None              |
+|   Remarks :             |  n/a               |
+
+## Objective
+
+Give the top 5 most played genres among games that have a number of distinct languages greater or equal to 3. 
+
+## Instructions
+
+You must write a query that will help you filter your games according to the number of languages they have, and then filter out the ones that have strictly less than 3 languages. Then you need to select the top 5 genres where those games appear.
```
Comment (madvid) [inline day00/ex12/ex12.md:16]: You must write a query ***which*** will help you **to** filter the games ...
At some point maybe you could ask to Amric to verify the english or any fluent english speaker

**Review by madvid — APPROVED** [review]
few tiny errors in some words,

File: day00/ex07/ex07.md (hunk 4 above)
Comment (fxbabin) [reply day00/ex07/ex07.md:12]: remark will be fixed in another PR

File: day00/ex03/ex03.md (hunk 2 above)
Comment (fxbabin) [reply day00/ex03/ex03.md:37]: remark will be fixed in another PR

File: day00/day00.md (hunk 1 above)
Comment (fxbabin) [reply day00/day00.md:9]: remark will be fixed in another PR

File: day00/ex07/ex07.md (hunk 5 above)
Comment (fxbabin) [reply day00/ex07/ex07.md:16]: remark will be fixed in another PR

File: day00/ex12/ex12.md (hunk 6 above)
Comment (fxbabin) [reply day00/ex12/ex12.md:16]: remark will be fixed in another PR

**Pull request merged**